lib/discourse_theme/scaffold.rb

```diff
@@ -1,75 +1,119 @@
 # frozen_string_literal: true
+
+require 'json'
+
 module DiscourseTheme
   class Scaffold
 
     BLANK_FILES = %w{
       common/common.scss
-      common/header.html
-      common/after_header.html
-      common/footer.html
-      common/head_tag.html
-      common/body_tag.html
-      common/embedded.scss
-
-      desktop/desktop.scss
-      desktop/header.html
-      desktop/after_header.html
-      desktop/footer.html
-      desktop/head_tag.html
-      desktop/body_tag.html
-
-      mobile/mobile.scss
-      mobile/header.html
-      mobile/after_header.html
-      mobile/footer.html
-      mobile/head_tag.html
-      mobile/body_tag.html
 
       locales/en.yml
 
       settings.yml
     }
 
-    ABOUT_JSON = <<~STR
-    {
-      "name": "#NAME#",
-      "about_url": null,
-      "license_url": null,
-      "assets": {
-      },
-      "color_schemes": {
-      }
+    ABOUT_JSON = {
+      about_url: nil,
+      license_url: nil,
+      assets: {}
     }
-  STR
 
     HELP = <<~STR
       Are you a bit lost? Be sure to read https://meta.discourse.org/t/how-to-develop-custom-themes/60848
     STR
 
     GIT_IGNORE = <<~STR
-    .discourse-site
-    HELP
-  STR
+      .discourse-site
+      node_modules
+      HELP
+    STR
+
+    API_INITIALIZER = <<~STR
+      import { apiInitializer } from "discourse/lib/api";
+
+      export default apiInitializer("0.11.1", api => {
+        console.log("hello world from api initializer!");
+      });
+    STR
+
+    PACKAGE_JSON = <<~STR
+      {
+        "author": "#AUTHOR",
+        "license": "MIT",
+        "devDependencies": {
+          "eslint-config-discourse": "latest"
+        }
+      }
+    STR
+
+    ESLINT_RC = <<~STR
+      {
+        "extends": "eslint-config-discourse"
+      }
+    STR
+
+    TEMPLATE_LINT_RC = <<~STR
+      module.exports = {
+        plugins: ["ember-template-lint-plugin-discourse"],
+        extends: "discourse:recommended",
+      };
+    STR
+
+    EN_YML = <<~STR
+      en:
+    STR
```

Comment:
```suggestion
    EN_YML = <<~YAML
      en:
        #my_locale_key: My Translation
        theme_metadata:
          description: "My theme description"
          settings:
            # my_setting_name: My Setting Description
    YAML
```
(maybe we could prompt for a description?)

Using `YAML` rather than `STR` means that it gets syntax-highlighted nicely in most editors (including GitHub UI)

Reply:
Cool stuff :+1: